CHANGELOG.rst

```diff
@@ -7,6 +7,16 @@ The format is based on `Keep a Changelog`_,
 and this project adheres to `Semantic Versioning`_.
 
 
+Version 2.2.0 (2024-07-23)
+--------------------------
+
+* Added: ``audeer.script_dir()``
+  which returns the directory of the file,
+  in which the function is called,
```

Comment:
I would say "  returns the directory of the file where it is called" or " returns the directory of the file from where it is called". I mean the "in which" that sounds strange to me.

Reply:
Good idea, I was also not happy with having two times "which". I changed to your first suggestion.